checks/security_policy_test.go

```diff
@@ -113,6 +113,16 @@ func TestSecurityPolicy(t *testing.T) {
 				NumberOfInfo: 1,
 			},
 		},
+		{
+			name: "pass_case_testing_case_insensitive",
```

Comment:
you can use spaces in the name, no ned for `_`

Reply:
Got it, thanks